### DISCUSSION TITLE
Not Available In My Country/Region

### DISCUSSION THREAD

**Discussion by willisplummer** [pixelspark/sushitrain#10] [Q&A] [answered]
Really excited to try this out after 6 months of using mobiussync, but it doesn't seem like I can install from the app store. I'm currently in Mexico, and I get the error "App Not Available - This app is currently not available in your country or region."

I wonder if there's a setting you need to change or something?

Thanks!

~

edit: just checked and my apple id country/region is set to the United States

**Comment by flochrislas** [comment]
Same here, I really wanted to use this app, but it's not availble in my country or region (Japan).

**Comment by Red6785** [comment]
Not in the United States, either.

**Comment by dhrm1k** [comment]
yes, @willisplummer. been facing the same problem. the app is not available in India. I have for now installed the testflight version.

**Comment by pixelspark** [comment, marked as answer]
More info here: https://t-shaped.nl/synctrain-support

In short, countries put restrictions or compliance requirements on importing software with cryptographic capabilities, and I don't have the time to figure them all out for each and every country. Assistance is appreciated and may help in making the app available in your country!
